src/Features/Core/Portable/UnusedReferences/Reference.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.CodeAnalysis.UnusedReferences
+{
+    internal class Reference
```

Comment:
so we already have MetadataReference.  I'm wary of reserving the name Reference for this type.  i'm strugglign with a better name though.  Maybe ReferenceDescription or ReferenceInfo.

Reply:
I like ReferenceInfo.